pkg/agents/team6/communication.go

```diff
@@ -254,6 +254,8 @@ func (a *CustomAgent6) HandleProposeTreaty(msg messages.ProposeTreatyMessage) {
 		a.Log("I accepted a treaty proposed from ", infra.Fields{"senderFloor": msg.SenderFloor(), "myFloor": a.Floor(), "my social motive": a.currBehaviour.string()})
 
 	} else {
+		reply := messages.NewTreatyResponseMessage(a.ID(), a.Floor(), msg.SenderFloor(), false, treaty.ID(), treaty.ProposerID())
+		a.SendMessage(reply)
 		a.updateTrust(-1, msg.SenderID()) // bad treaty - these guys are trying to sabotage us >:)
```

Comment:
Why would you not use `proposeTreatyMessage.Reply(...)`?

Reply:
Good spot! We did not know about the existence of the `proposeTreatyMessage.Reply()` function. We just used the less direct way by creating a new `NewTreatyResponseMessage` by ourself